packages/sidenav/test/sidenav.test.ts

```diff
@@ -393,6 +394,11 @@ describe('Sidenav', () => {
     it('focuses the child anchor not the root when [tabindex=-1]', async () => {
         const el = await fixture<SideNav>(manageTabIndex());
 
+        await nextFrame();
+        await nextFrame();
+        await nextFrame();
+        await nextFrame();
```

Comment:
Why all the nextFrames here?

Reply:
This is working to ensure that all child elements have `el.updateComplete`'ed. We would likely benefit from an advanced version of `fixture` or other helper that ensured whole tree of elements has updated before running tests.